build.sh

```diff
@@ -0,0 +1,10 @@
+#! /bin/bash
+
+set -e
+set -x
+
+mkdir -p build
+cp -R planet/* build/
```

Comment:
Don't know, you can try to remove it. I think it copies some important files that we want to be part of the repository.